lib/utils/FileCache.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+"use strict"
+
+const fileEntryCache = require("file-entry-cache")
+const path = require("path")
+const debug = require("debug")("stylelint:file-cache")
+const getCacheFile = require("./getCacheFile")
+const DEFAULT_CACHE_LOCATION = "./.stylelintcache"
```

Comment:
Let's add an empty line above this constant to differentiate it from the dependencies.